test/test.go

```diff
@@ -414,6 +417,47 @@ func (h *Helper) TempFile(path, contents string) {
 	h.Must(ioutil.WriteFile(filepath.Join(h.tempdir, path), bytes, 0644))
 }
 
+// GetTestFileBytes reads a file form the testdata directory into memory.  src is
+// relative to ./testdata.  Assumes tests take place starting in the cmd/dep
+// directory.
+func (h *Helper) GetTestFileBytes(src string) []byte {
+	content, err := ioutil.ReadFile(filepath.Join(h.origWd, "_testdata", src))
+	if err != nil {
+		panic(err)
+	}
+	if strings.HasSuffix(src, ".go") {
+		formatted, err := format.Source(content)
```

Comment:
We should be writing well formatted code to disk when necessary and not reformatting it. This can lead to issues when tests will need to deal with poorly formatted code.